test/unit/alignment/configuration/align_config_common_test.cpp

```diff
@@ -26,7 +26,8 @@ using test_types = ::testing::Types<align_cfg::aligned_ends<std::remove_const_t<
                                     align_cfg::mode<detail::global_alignment_type>,
                                     align_cfg::mode<detail::local_alignment_type>,
                                     align_cfg::result<>,
-                                    align_cfg::scoring<nucleotide_scoring_scheme<int8_t>>>;
+                                    align_cfg::scoring<nucleotide_scoring_scheme<int8_t>>,
```

Comment:
The place of vectorise would also change here.

Reply:
As stated above, this determines just the order of the tests being executed and has been established to an alphabetical order...